PeerioTechnologies/peerio-desktop
# style: highlight AppNav item of the currently selected view

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/827/desktop-settings-section-in-drow-down-menu

#### Testing instructions  
When you're on any Settings view, the view that you currently have open should be highlighted blue in the AppNav menu.


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
